samir-koirala

```diff
@@ -0,0 +1 @@
+Samir Koirala
```

Comment:
<html>
<head> <title> Samir Koirala </title> </head>
<body>
<h1>
Samir Koirala
</h1>
<p>
Executive member of OSAC
</p>
</body>
</html>